src/components/certificates/NewCertificateForm.tsx

```diff
@@ -0,0 +1,195 @@
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { ReactComponent as CloseSVG } from '../../assets/images/close.svg';
+import { ReactComponent as SearchSVG } from '../../assets/images/search.svg';
+import { addNewCertificate } from '../../data/db';
+import { CertificateType } from '../../types/types';
+import InputField from '../Form/InputField';
+import SelectField from '../Form/SelectFIeld';
+import SVGIcon from '../SVGIcon/SVGIcon';
+import './certificateForm.css';
+
+const NewCertificateForm: React.FC = () => {
+  const navigate = useNavigate();
+
+  const initialFormState = {
+    supplier: '',
+    certificateType: CertificateType.OHSAS18001,
+    validFrom: '',
+    validTo: '',
+    PDFUrl: '',
+  };
+
+  const [formState, setFormState] = useState(initialFormState);
+  const [formError, setFormError] = useState<string>('');
+  const [PDFUrl, setPDFUrl] = useState<string | null>(null);
```

Comment:
Not sure why zou have PDF URL as separate state when it is already in form state? Can you just use the same from the form